components/ca-radio-group.js

```diff
@@ -66,26 +70,26 @@ define([], () => {
 
         /**
          * Sets data
-         * @param {array} data data
+         * @param {object} data data
          * @returns {void} void
          */
         set data(data) {
```

Comment:
If 'data' is actually a property schema, can we call it something more meaningful?